src/raftstore/store/msg.rs

```diff
@@ -162,10 +163,10 @@ pub enum Msg {
         hash: Vec<u8>,
     },
 
-    // For region size
-    ApproximateRegionSize {
+    // For region stat
+    ApproximateRegionStat {
```

Comment:
ApproximateRegionStat and RegionApproximateStat, oh, they will cause confusing.

Reply:
Can we use `RegionApproximateStat` here too? Will it conflict?